releasenotes/notes/support_excitedstatessolver_bopes-2bbb5a282b1ff582.yaml

```
@@ -0,0 +1,21 @@
+features:
+  - |
+    The solver of BOPESSampler now accepts solvers of type
+    `Union[GroundStateSolver,ExcitedStatesSolver]` instead of only a `GroundStateSolver`. 
+    This generalizes to excited states the sampling 
+    of nuclear coordinates for Born Oppenheimer Potential Energy Surfaces.
+    Adds the `get_qubit_operators()` in
+    :class:`qiskit_nature.algorithms.ExcitedStatesEigensolver`.
+    This matches the format of
+    :class:`qiskit_nature.algorithms.GroundStatesEigensolver`.
```

```suggestion
    :class:`qiskit_nature.algorithms.GroundStateEigensolver`.
```
No `s` in the ground state one (it was failing to resolve this in the outputs docs and it was not clickable as a link).

---

done, thanks